doc/explanation/microcloud.rst

```diff
@@ -24,16 +24,21 @@ See :ref:`lxd:clustering` in the LXD documentation for more information.
 Networking
 ----------
 
-.. include:: ../../README.md
-   :parser: myst_parser.sphinx_
-   :start-after: ## **What about networking?**
-   :end-before: ## **What's next?**
+By default, MicroCloud uses MicroOVN for networking, which is a minimal implementation of OVN (Open Virtual Network).
```

Comment:
I don't think we should say its a minimal implementation of ovn. How about a "minimal alternative to OVN"?

Reply:
Changed to "minimal wrapper around OVN".

Reply:
No that isn't correct I'm afraid, as Ubuntu FAN isn't anything to do with OVN.

Reply:
Hmm, not sure I'm following - we're talking about MicroOVN here and not about Ubuntu Fan?